pages.fr/common/ls.md

```diff
@@ -0,0 +1,27 @@
+# ls
+
+> Liste le contenu d'un répertoire.
+
+- Affiche un fichier par ligne:
+
+`ls -1`
+
+- Affiche tous les fichiers, ainsi que les fichiers cachés:
+
+`ls -a`
+
+- Affiche tous les fichiers avec un format détaillé (permissions, propriétaire, taille et date de modification):
+
+`ls -la`
+
+- Affiche les fichiers avec un format détaillé. Les tailles utilisent des prefixes d'unités pour améliorer la lisibilité (KB, MB, GB):
+
+`ls -lh`
+
+- Liste les fichier avec un format détaillé. Les fichers sont triés par taille decroissante:
```

Comment:
Again, it would be nice to make this line shorter, perhaps by combining the sentences.